tests/bindings/python/test_model.py

```diff
@@ -206,9 +206,9 @@ def test_replace_units(self):
         a = Units()
         m.addUnits(a)
         b = Units()
-        self.assertFalse(m.replaceUnits(b, a))
-        self.assertFalse(m.replaceUnits(a, b))
-        self.assertFalse(m.replaceUnits(b, a))
+        self.assertTrue(m.replaceUnits(b, a))
```

Comment:
I think this test needs to be made a lot clearer.  This is a weird case where `replaceUnits` takes arguments of old, new -> b doesn't exist in the model, so nothing should happen.  BUT since both are empty, the `equal` function will assume that the user means replace pseudo_b (ie: a) with itself ... so the references are switched, but the content is not?  It would demonstrate the behaviour more clearly if there was identical non-empty content in both units; as it is, this is just confusing and counter-intuitive.

Reply:
I don't see how having identical non-empty content changes anything equal is still equal and the replacement would still return `True`.  Again the Python tests are for exercising the Python API functional examples should be made in the C++ tests.